swagger.yaml

```diff
@@ -905,7 +956,15 @@ definitions:
         type: string
       yearReceived:
         type: string
-  PersonalObject:
+  PersonalResultObject:
+    properties:
+      links:
+        $ref: "#/definitions/ResultLinks"
+      data:
```

Comment:
Can a person have more than one of this? Meaning should this be an array or a single object?